application/frontend/src/pages/Search/components/SearchBar.scss

```
@@ -13,35 +13,31 @@
   }
+
   input {
+    display: block;
     padding: 0.5rem 1rem 0.5rem 2.5rem;
     width: 100%;
     max-width: 16rem;
 
-    background-color: rgba(var(--card-rgb), 0.5); /* tiny bump */
-    border: 1px solid rgba(102, 157, 246, 0.267); /* slightly brighter sky blue */
-
+    background-color: rgba(255, 255, 255, 0.04);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)